product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters.mdx

```diff
@@ -6,15 +6,45 @@ The database parameters listed on the DB Configuration tab are also referred to
+- For PostgreSQL parameters, see [Setting Parameters](https://www.postgresql.org/docs/current/config-setting.html) and [Server Configuration](https://www.postgresql.org/docs/current/runtime-config.html) in the Postgres documentation.
      
 
--   For EDB Postgres Advanced Server, see [Summary of Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/02_summary_of_configuration_parameters/#summary_of_configuration_parameters) and [Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/).
+- For EDB Postgres Advanced Server, see [Summary of Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/02_summary_of_configuration_parameters/#summary_of_configuration_parameters) and [Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/).
+
+-  For more information on the types of values parameters accept, see [Parameter Names and Values](https://www.postgresql.org/docs/current/config-setting.html#CONFIG-SETTING-NAMES-VALUES). The types apply to both PostgreSQL and EDB Postgres Advanced Server parameters.
+
 
 !!!note
 Not all database configuration parameters are supported by BigAnimal. Some parameters, such as `wal_level` and `restore_command`, are reserved for EDB to provide the managed database features of BigAnimal. 
 !!!
 
+## Using formulas for parameter values
+
+In addition to entering specific values for parameters, for some parameters you can specify formulas to automatically calculate a value. You can use formulas for parameters of type integer and real using the following operators: `+ - / * > >= < <= == != && || ! ? : ( )`. GUCs used in formulas must also be of type integer or real. 
+
+BigAnimal has what we refer to as psuedo GUCs to facilitate creating equations. These read-only GUCs are:
+- `cpu_cores`
+- `disk_iops`
+- `disk_size`
+- `ram`
+- `replica_count` (node count)
+
+`ram` and `disk_size` have the default unit of KB. 
+
+You can't use GUCs with different types of units (such as size or duration) in formulas. For example, you can't use a formula that tries to multiply two seconds with three kilobytes (2s \* 3KB), but you can use a formula that multiplies two sizes even if they have different scales (2KB \* 8GB).
+
+Formulas can calculate a value relative to the specifications you selected for the cluster or relative to other non-pseudo GUCs. If you resize your cluster, BigAnimal recalculates formulas using the new values for the pseudo GUCs. 
+
+
+### Example 
+
+This example uses a formula for the `shared_buffers` parameter value.
```

Comment:
The fact that this is for `shared_buffers` is somewhat irrelevant, right? It's useful for the description but meaningless elsewhere. How about we construct a somewhat more complex example, e.g., set shared buffers to the formula shown here and then set another GUC to a formula that includes `shared_buffers`?

Reply:
We might also want to demonstrate and explain some common errors, e.g., type errors?

Reply:
@edbmanniwood Ben has suggested we add more examples. Can you help with that?

Reply:
The parser will let users know about malformed equations. For instance, following a number with two units will result in an error.

work_mem = 10 GB kB

produces the error

unexpected token "kB" at end of input (line 1, col 7)

A GUC referring to itself would set up an infinite evaluation loop, so it is disallowed.

effective_cache_size = effective_cache_size

gives the error

cycle in equation;  GUC "effective_cache_size" already seen

Mixing unit types is always a bad idea. The GUC evaluator does not know what the result of 10 kB * 3 s would be.

work_mem = 10kB * 8s

gives the error

both size and time units used in calculation; cannot proceed.

Reply:
Thanks, @edbmanniwood. @ba-edb and Manni, how does this look? https://github.com/EnterpriseDB/docs/pull/2361/commits/00ed71215b078cd23944e3b6a48641699d848fc1

Reply:
Hey, @drothery-edb . I left a couple of quick comments. Nothing big. Thanks for your persistence! :bow: